install.sh

```diff
@@ -118,20 +155,21 @@ say "Target:      $target"
 say "Destination: $dest"
 say "Archive:     $archive"
 
-tempdir=`mktemp -d || mktemp -d -t tmp`
-
-curl --proto =https --tlsv1.2 -sSfL $archive | tar --directory $tempdir --strip-components 1 -xz
+td=$(mktemp -d || mktemp -d -t tmp)
 
-for name in `ls $tempdir`; do
-  file="$tempdir/$name"
-  test -x $file || continue
+if [ "$extension" = "zip" ]; then
```

Comment:
this code suggestion eliminates the 404 error I was running into on Windows when running `./install.sh --force` with the fix from this branch.

```suggestion

if [ "$extension" = "zip" ]; then
  download "$archive" "$td/ord.zip"
  unzip -d "$td" "$td/ord.zip"
  binary_path=$(find "$td" -type f \( -name "ord.exe" -o -name "ord" \) | head -n 1)
else
  download "$archive" - | tar --directory "$td" --strip-components 1 -xz
  binary_path=$(find "$td" -name ord | head -n 1)
fi

if [ -z "$binary_path" ]; then
  err "Binary not found in the archive"
fi

binary_name=$(basename "$binary_path")

if [ -e "$dest/$binary_name" ] && [ "$force" = false ]; then
  err "\`$dest/$binary_name\` already exists"
else
  mkdir -p "$dest"
  cp "$binary_path" "$dest/$binary_name"
  chmod 755 "$dest/$binary_name"
fi

rm -rf "$td"
```

Reply:
the error I was hitting was as follows:
```bash
Archive:  /tmp/tmp.yF7M2GQnOS/ord.zip
   creating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/
  inflating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/Cargo.lock
  inflating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/Cargo.toml
  inflating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/LICENSE
  inflating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/ord.exe
  inflating: /tmp/tmp.yF7M2GQnOS/ord-0.22.1/README.md
cp: cannot stat '/tmp/tmp.yF7M2GQnOS/ord.exe': No such file or directory
```